mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -1164,7 +1164,7 @@ struct GridwiseGemmV2RewritePattern
     }
 
     // Obtain critical tuning parameters.
-    uint32_t blockSize = op.getBlockSize();
+    uint32_t blockSize = op.getDerivedBlockSize();
```

Comment:
GridwiseGemmV2 should not have experienced this renaming, it's on the xdlops path